src/main/java/fr/quatrevieux/araknemu/game/fight/turn/action/move/Move.java

```diff
@@ -54,9 +64,55 @@ public boolean validate() {
         ;
     }
 
+    private MoveStatus handleTackle() {
```

Comment:
Use a path validator system, like in https://github.com/Arakne/Araknemu/blob/master/src/main/java/fr/quatrevieux/araknemu/game/exploration/interaction/action/move/Move.java#L56